nomad/node_pool_endpoint.go

```diff
@@ -252,3 +253,128 @@ func (n *NodePool) DeleteNodePools(args *structs.NodePoolDeleteRequest, reply *s
 	reply.Index = index
 	return nil
 }
+
+// ListJobs is used to retrieve a list of jobs for a given node pool. It supports
+// pagination and filtering.
+func (n *NodePool) ListJobs(args *structs.NodePoolJobsRequest, reply *structs.NodePoolJobsResponse) error {
+	authErr := n.srv.Authenticate(n.ctx, args)
+	if done, err := n.srv.forward("NodePool.Jobs", args, args, reply); done {
```

Comment:
doesn't this need to be `NodePool.ListJobs` ?

Reply:
oof, yes.

Reply:
Fixed